helm/reana/templates/reana-server.yaml

```diff
@@ -54,10 +54,13 @@ spec:
           - mountPath: {{ .Values.shared_storage.shared_volume_mount_path }}
             name: reana-shared-volume
           {{- range $workspace_paths := .Values.workspaces.paths }}
-          {{- if not (eq $workspace_paths "/var/reana")}}
-          - name: {{ $workspace_paths | quote |replace "/" "" }}
-            mountPath: {{ $workspace_paths }}
-            mountPropagation: HostToContainer
+          {{- if eq (typeOf $workspace_paths) "map[string]interface {}" -}}
```

Comment:
We could use `{{- if ne (typeOf $workspace_paths) "string" -}}` here, I guess it would be more clear approach